core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -49,6 +49,32 @@ import scala.collection.{Seq, mutable}
 class KafkaZkClient(zooKeeperClient: ZooKeeperClient, isSecure: Boolean) extends Logging {
   import KafkaZkClient._
 
+  def createSequentialPersistentPath(path: String, data: String = ""): String = {
```

Comment:
Could we add a unit test for the new methods added in this class in KafkaZkClientTest?

Reply:
Seems like this still needs to be addressed.